tests/Makefile

```diff
@@ -4,6 +4,7 @@ export CFLAGS+=-g -O0 -Wall -D_GNU_SOURCE
 DISTRO = $(shell ./os_detect)
 
 SUBDIRS := \
+	exclude_filter \
```

Comment:
I'm trying to see what naming convention was already used.  It appears to be <object>_<verb>, but some are compound objects.  What is the convention you see or understand?